### PR TITLE
Queues with stacks

### PR DESCRIPTION
**Pull request by TiaLow**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution
    - [x] Link to code
    - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/challenges/queuesWithStacks/queue-with-stacks.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    const valOfPopped = this.stack2.pop();
+
+    while(!this.stack2.isEmpty()){
+      let itemPopped = this.stack2.pop();
+      this.stack1.push(itemPopped);
+      this.length++;
+    }
+
```
Comment (adrianhuebner) [inline javascript/challenges/queuesWithStacks/queue-with-stacks.js:38]: Something you could do for less code would be to check if stack1 is empty or not, throw the error that it is. Then push into stack2, popping off of stack1 (hope that makes sense) and then pop off stack2. Because when you push from stack1 to stack2 it will reverse the order!
